howto/dashboard/web-dashboard.md

```diff
@@ -27,7 +27,7 @@ Before you can log into the dashboard, you must register your Ubuntu One account
 
 On a regular Anbox Cloud deployment, use the following Juju action to register an Ubuntu One account:
 
-    juju run-action anbox-cloud-dashboard/0 --wait register-account email=<Ubuntu One email address>
+    juju run anbox-cloud-dashboard/0 --wait register-account email=<Ubuntu One email address>
```

Comment:
When using the `juju run` command, it is necessary to specify a specific timeout value. For example:
`juju run anbox-cloud-dashboard/0 --wait=5m register-account email=<Ubuntu One email address>`
otherwise an error occurs. E.g.
```
ERROR invalid value "register-account" for option --wait: time: invalid duration "register-account"
```

Same below

Reply:
Added for all `juju run` mentions.